pkg/syfs/syfs.go

```diff
@@ -68,3 +69,9 @@ func ConfigDirForUsername(username string) (string, error) {
 
 	return filepath.Join(u.HomeDir, singularityDir), nil
 }
+
+// GetSysConfigFile returns the path the Singularity system
+// configuration file
```

Comment:
No period at the end of documentation comment.

Reply:
done.